vnext/Microsoft.ReactNative/Fabric/ComponentView.h

```diff
@@ -0,0 +1,48 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#pragma warning(push)
+#pragma warning(disable : 4244)
+#include <react/renderer/componentregistry/ComponentDescriptorProvider.h>
+#include <react/renderer/core/LayoutMetrics.h>
+#pragma warning(pop)
+
+namespace Microsoft::ReactNative {
+
+using RNComponentViewUpdateMask = std::uint_fast8_t;
+/*
+ * Bitmask for all types of possible updates performed during mounting.
+ */
+constexpr RNComponentViewUpdateMask RNComponentViewUpdateMaskNone{0};
```

Comment:
+1 to nit about doing an `enum class` for these so we can get better type safety.